streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/SessionWindowedCogroupedKStream.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.scala
+package kstream
+
+import org.apache.kafka.streams.kstream.{
+  SessionWindowedCogroupedKStream => SessionWindowedCogroupedKStreamJ,
+  _
+}
+import org.apache.kafka.streams.scala.FunctionsCompatConversions._
+import org.apache.kafka.streams.scala.ImplicitConversions._
```

Comment:
IntelliJ complains that this is an unused import, but without it, it does not compile at command line -- what might I do wrong?

Reply:
You can explicitly import the implicit conversions that you're using. Actually, I would recommend not even using them implicitly, but just to explicitly invoke the relevant conversion methods when you need them. (i.e., implicit conversions are syntactic sugar for users of the public API, we should try to instead write explicit and maintainable code in our internal implementations)

It should be pretty obvious which ones you need if you just delete this line and see what fails to compile.

Reply:
Based on this commend, I did a proper cleanup PR: https://github.com/apache/kafka/pull/7852

We should merge the cleanup PR first, and I can rebase this PR afterwards.